app/views/account/sign-in-to-another-service.html

```diff
@@ -83,9 +83,9 @@ <h3 class="govuk-heading-m  govuk-!-padding-top-2 govuk-!-padding-bottom-2"><a h
 </a></h3><p>
 Apply for rural grants and payments.
 </P>
-<h3 class="govuk-heading-m  govuk-!-padding-top-2 govuk-!-padding-bottom-2"><a href="">
+<h3 class="govuk-heading-m  govuk-!-padding-top-2 govuk-!-padding-bottom-2">
```

Comment:
There's a bit of formatting stuff awry in this file, I'll merge it and then run it through a linter?